src/server/views/layout.html

```diff
@@ -175,21 +175,21 @@ <h1 class="govuk-heading-l">Default page template</h1>
 {% endblock %}
 
 {% block footer %}
-    {{ govukFooter({
-        meta: {
-            items: [{
-                href: '/help/get-support' + ('/' + slug if slug),
-                text: 'Get help with this form'
-            }, {
-                href: '/help/privacy' + ('/' + slug if slug),
-                text: 'Privacy'
-            }, {
-                href: '/help/cookies',
-                text: 'Cookies'
-            }, {
-                href: '/help/accessibility-statement',
-                text: 'Accessibility Statement'
-            }]
-        }
-    }) }}
+  {% set meta = {
+    items: [{
+      href: '/help/get-support/' + slug,
+      text: 'Get help with this form'
+    }, {
+      href: '/help/privacy/' + slug,
+      text: 'Privacy'
+    }, {
+      href: '/help/cookies/' + slug,
+      text: 'Cookies'
+    }, {
+      href: '/help/accessibility-statement/' + slug,
+      text: 'Accessibility Statement'
+    }]
+  } if slug %}
+
+  {{ govukFooter({ meta: meta }) }}
```

Comment:
I know we don't have Prettier for Nunjucks, but want to format these squished meta items?

Each object on their own line etc

```suggestion
  {{ govukFooter({
    meta: {
      items: [
        {
          href: '/help/get-support/' + slug,
          text: 'Get help with this form'
        },
        {
          href: '/help/privacy/' + slug,
          text: 'Privacy'
        },
        {
          href: '/help/cookies/' + slug,
          text: 'Cookies'
        },
        {
          href: '/help/accessibility-statement/' + slug,
          text: 'Accessibility Statement'
        }
      ]
    } if slug
  }) }}
```